controller/core/src/keys.rs

```diff
@@ -41,6 +41,12 @@ id_str!(
     "Sysvar1nstructions1111111111111111111111111"
 );
 
+id_str!(
+    INF_MINT_ID_STR,
+    INF_MINT_ID,
+    "5oVNBeEEQvYi1cX3ir8Dx5n1P7pdxydbGF2X4TxVusJm"
+);
```

Comment:
think we can put this in `test-utils` if its just for testing. The program is sposed to work with the mint stored in `pool_state.lp_token_mint`